static/app/stores/groupingStore.spec.tsx

```diff
@@ -241,6 +261,53 @@ describe('Grouping Store', () => {
       });
     });
 
+    it('handles fingerprints with metadata including seer information', () => {
+      const promise = GroupingStore.onFetch([
+        {dataKey: 'merged', endpoint: '/issues/groupId/hashes/'},
+      ]);
+
+      expect(trigger).toHaveBeenCalled();
+      const calls = trigger.mock.calls;
+      return promise.then(() => {
+        const arg = calls[calls.length - 1][0];
+        const mergedItems = arg.mergedItems;
```

Comment:
instead of looking at "arg" here maybe we can call getState which should be the same object and has proper types